cluster/gce/config-test.sh

```diff
@@ -328,6 +328,20 @@ else
   ADMISSION_CONTROL=${KUBE_ADMISSION_CONTROL}
 fi
 
+if [[ ${ADMISSION_CONTROL} == *"Initializers"* ]]; then
+    if [[ -n "${RUNTIME_CONFIG}" ]]; then
+      RUNTIME_CONFIG+=","
+    fi
+    RUNTIME_CONFIG+="admissionregistration.k8s.io/v1alpha1"
+fi
+
+if [[ ${ADMISSION_CONTROL} == *"AdmissionWebhook"* ]]; then
+    if [[ -n "${RUNTIME_CONFIG}" ]]; then
+      RUNTIME_CONFIG+=","
+    fi
+    RUNTIME_CONFIG+="admissionregistration.k8s.io/v1beta1"
```

Comment:
See https://github.com/kubernetes/kubernetes/pull/56644. It's a bug that we haven't enabled it by default.

Reply:
@caesarxuchao Should this pr waiting for #56644 merged ?

Reply:
BTW, I like this modification.

Reply:
https://github.com/kubernetes/kubernetes/pull/56687 is merged. We can remove this block now.

Reply:
ok